src/current/cockroachcloud/authorization.md

```diff
@@ -108,15 +117,45 @@ Cluster Administrators can perform all of the [Cluster Operator actions](#cluste
 - Cluster Administrators for the whole organization (rather than scoped to a single cluster) can [create new clusters](create-your-cluster.html).
 - Access the [DB Console](network-authorization.html#db-console).
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
 ### Cluster Creator
 
 Cluster Creators can create clusters in an organization. A cluster's creator is automatically granted the [Cluster Administrator](#cluster-administrator) role for that cluster upon creation.
 
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
 ### Cluster Developer
 
-Users with this role can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
+Cluster Developers can view cluster details and access the [DB Console](network-authorization.html#db-console), allowing them to [export a connection string from the cluster page UI](authentication.html#the-connection-string), although they will still need a Cluster Administrator to [provision their SQL credentials](managing-access.html#manage-sql-users-on-a-cluster) for the cluster.
+
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
+
+### Folder Admin
+
+{% include_cached feature-phases/limited-access.md %}
 
-This role can be granted for specific clusters or for all clusters in the organization.
+This role is available only when your organization is enrolled in the Folders Limited Access.
+
+Folder Admins can create, rename, and move, or delete folders where they are granted the role, and they can also manage access to these folders. This role can be granted at the level of the organization or on a specific folder. If granted on a specific folder, the role is inherited by descendent folders.
+
+A user with the [Org Administrator](#org-administrator) role can grant themselves, another user, or a service account the Folder Admin role.
```

Comment:
nit: service accounts are users so maybe something like "this role can be granted to any user (person, service account, or group)"

Reply:
I don't want to use the parenthetical so I'll iterate your suggestion a bit.